pkg/util/informermanager/testutils.go

```diff
@@ -0,0 +1,216 @@
+package informermanager
```

Comment:
Should we move test utils to a separate package so that we do not compile this file when not building unit tests?